raspberry/checkout/checkout_main.py

```diff
@@ -62,16 +65,40 @@ def process_rfid_card(self, uid):
             print("Item already scanned")
             self.interactions.buzz()
             self.interactions.indicate_error()
```

Comment:
cant put this comment any higher cuz GH is weird, but in the function process_rfid_card, we should first check, if state is 0. we don't want to be able to scan new items in the summary. This can pretty much screw up our logic and ordering. so just do something like if self.state == 1: return at the beginning of the function